assets/src/scripts/monitoring-location/selectors/daily-value-time-series-selector.spec.js

```diff
@@ -477,5 +478,21 @@ describe('monitoring-location/selectors/daily-value-time-series-selector', () =>
                 max: 14.0
             });
         });
+
+        it('Should return true if there are multiple parameter codes in available', () => {
+            expect(hasMultipleParameterCodes(
+                {dailyValueTimeSeriesData:
+                        {availableDVTimeSeries:
+                            [{parameterCode: '1'}, {parameterCode: '2'}]
+                        }})).toBeTruthy();
+        });
+
+        it('Should return true if there are multiple parameter codes in available', () => {
```

Comment:
Typo in text, should be "Should return _false_"

Reply:
I changed the text to say false.